### DISCUSSION TITLE
Query: wiki docs

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1303] [Q&A] [answered]
![ex](https://imgur.com/SkGMvzR.png)

https://github.com/BlitterStudio/amiberry/wiki/WHDLoad-Auto-booting#game-specific-settings-improving-game-compatibility

Shouldn't that be the  `Configurations`  path (not whdboot/ )? 

TIA

**Comment by midwan** [comment, marked as answer]
That's correct, I'll fix it. Thanks for noticing!

**Comment by giantclambake** [comment]
All good, thanks ;)
